packaging/debian-sid/snapd.postinst

```diff
@@ -36,6 +36,12 @@ case "$1" in
         fi
         # Ensure that the void directory has correct permissions.
         chmod 111 /var/lib/snapd/void
+
+        # TODO update caps
```

Comment:
Not sure what you mean here. Is not `postinst configure` also called on upgrade?

Reply:
rephrased the comment